docs/getting_started/bgen/runscript.md

```diff
@@ -157,21 +157,23 @@ jobs (`distjob`) for running benchmarks on a cluster.
 ### Sequential Jobs
 
 A sequential job is identified by its `name` and sets the `timeout` (in
-seconds) for a single run, the number of `runs` for each instance, and the
-number of solver processes performed in `parallel`:
+seconds) for a single run, the number of `runs` for each instance, and
+the number of solver processes performed in `parallel`. The optional
```

Comment:
Maybe consider using "executed in parallel" or "run in parallel" - "performed" sounds weird to me.